crates/iota-grpc-api/README.md

```diff
@@ -0,0 +1,50 @@
+# IOTA gRPC API
+
+> **⚠️ EXPERIMENTAL - INTERNAL USE ONLY**
+>
+> This gRPC API is highly experimental and intended for internal use only. The API surface, data formats, and behavior are subject to significant changes without notice. **Do not use this in production or build external integrations against it** as breaking changes are expected and likely.
+
+This crate introduces a gRPC API for IOTA. The primary goal of this API is to provide a more efficient and lower-latency method for data access, intended to replace existing REST-API polling or filesystem-based synchronization. This reduces the delay between data creation and their subsequent processing by external services.
+
+## Features
+
+The `NodeService` provides the following RPC endpoints:
+
+- `StreamCheckpoints`: Stream checkpoint data based on a flexible range.
+- `GetEpochFirstCheckpointSequenceNumber`: Query the first checkpoint sequence number for a given epoch (useful for robust reset and epoch boundary handling).
+
+## Usage
+
+The `iota-grpc-api` crate defines the gRPC service and its messages. The `iota-node` crate integrates and starts this gRPC server if `enable_grpc_api` is set to `true` and `grpc_api_config` is configured.
```

Comment:
In the config the keys are normally not snake_case, so this should be `enable-grpc-api`, etc.